modules/ROOT/pages/values-and-types/maps.adoc

```diff
@@ -0,0 +1,156 @@
+:description: This section describes how to use maps in Cyphers.
+
+[[cypher-maps]]
+= Maps
+
+Cypher supports the construction of maps.
+This section first discusses literal maps and then moves on to map projection.
+
+====
```

Comment:
This box looks a bit strange when rendered for me (has a big space under the words), is that correct?